cmd/storage-rest-client.go

```diff
@@ -332,6 +332,7 @@ func (client *storageRESTClient) CheckParts(volume, path string, fi FileInfo) er
 
 	var reader bytes.Buffer
 	if err := gob.NewEncoder(&reader).Encode(fi); err != nil {
+		logger.LogIf(context.Background(), err)
```

Comment:
is it necessary? this may add spurious logs.

Reply:
It never should unless something is messed up, right?

Reply:
In-cases, when files are missing it, would spuriously log which is not a big deal to be present.

Reply:
That shouldn't make the encode fail. Only if fields are added that GOB cannot encode AFAICT,